tensorflow/python/BUILD

```diff
@@ -2524,6 +2524,18 @@ py_library(
     ],
 )
 
+py_test(
```

Comment:
Delete this build rule and instead add your tests to tensorflow/python/kernel_tests/py_func_test.py

Reply:
Can do - thanks for the pointers. Will fix and push shortly.